### PR TITLE
feat(ReportsNew): add domains and date filters

### PR DESCRIPTION
**Pull request by fcoronelmakingsense**
- Add filters in the new reports page.
- Add the posibility to select 2 or 3 weeks in the new reports page.

image

### CODE REVIEW

File: src/components/Reports/ReportsFilters/ReportsFilters.js [hunk 1]
```diff
@@ -89,19 +90,22 @@ const ReportsFilters = ({
...
-                    >
-                      {(message) => <option value="14">{message}</option>}
-                    </FormattedMessage>
-                    <FormattedMessage
-                      id="reports_filters.week_with_plural"
-                      values={{ weeksCount: 3 }}
-                    >
-                      {(message) => <option value="21">{message}</option>}
-                    </FormattedMessage>*/}
+                    {isEnableWeeks ? (
```
Comment (andresmoschini) [inline src/components/Reports/ReportsFilters/ReportsFilters.js:93]: ❤️

Comment (andresmoschini) [reply src/components/Reports/ReportsFilters/ReportsFilters.js:93]: Do you think that adding a test for this parameter could make sense?

**Comment by cbernat** [comment]
👍

File: src/components/Reports/ReportsFilters/ReportsFilters.js (hunk 1 above)
Comment (fcoronelmakingsense) [reply src/components/Reports/ReportsFilters/ReportsFilters.js:93]: yes i added now!

File: src/components/Reports/ReportsFilters/ReportsFilters.test.js [hunk 2]
```diff
@@ -61,4 +61,32 @@ describe('ReportsFilters component', () => {
     // Assert
     expect(container).not.toContainHTML('reports_filters.verified_domain');
   });
+
+  it('should show 2 and 3 weeks', async () => {
```
Comment (andresmoschini) [inline src/components/Reports/ReportsFilters/ReportsFilters.test.js:65]: What do you think about also adding the other scenario?

```js
   it('should show 1 week by default', async () => {
```

Comment (andresmoschini) [inline src/components/Reports/ReportsFilters/ReportsFilters.test.js:65]:
```suggestion
  it('should show 2 and 3 weeks when isEnableWeeks', async () => {
```

File: src/components/Reports/ReportsFilters/ReportsFilters.js (hunk 1 above)
Comment (andresmoschini) [reply src/components/Reports/ReportsFilters/ReportsFilters.js:93]: 🎉

File: src/components/Reports/ReportsFilters/ReportsFilters.test.js (hunk 2 above)
Comment (fcoronelmakingsense) [reply src/components/Reports/ReportsFilters/ReportsFilters.test.js]: yes i added now! thanks

File: src/components/Reports/ReportsFilters/ReportsFilters.test.js [hunk 3]
```diff
@@ -61,4 +61,59 @@ describe('ReportsFilters component', () => {
...
+          changePeriod={() => {}}
+        />
+      </DopplerIntlProvider>,
+    );
+
+    // Assert
+    expect(getAllByText('reports_filters.week_with_plural'));
+  });
+
+  it('should show 1 week when isEnableWeeks false', async () => {
```
Comment (andresmoschini) [inline src/components/Reports/ReportsFilters/ReportsFilters.test.js:93]: ❤️

**Pull request merged**